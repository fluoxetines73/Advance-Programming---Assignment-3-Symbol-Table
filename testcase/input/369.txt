INSERT f_FF string
INSERT y string
ASSIGN f_FF 'HelloWorld'
RPRINT
PRINT
ASSIGN f_FF 'Test123'
ASSIGN y 'UPPERlower'
PRINT
BEGIN
PRINT
ASSIGN y 987654
ASSIGN f_FF '1234ABC'
END