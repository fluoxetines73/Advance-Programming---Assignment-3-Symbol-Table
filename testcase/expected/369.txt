TypeMismatch: ASSIGN y 987654